tests/Refinery/String/MakeClickableTest.php

```diff
@@ -0,0 +1,58 @@
+            'test multiple + nested + multibyte' => ['Praesent fermentum tempor ♥ tellus <span>Aliquam erat volutpat<b><a href="www.ilias.de"><b>curabitur lacinia pulvinar nibh</b> www.ilias.de</a>nunc eleifend leo vitae magnawww.ilias.denunc rutrum turpis sed pede <a href="https://www.ilias.de">www.ilias.de</a> phasellus ♥ diam<a href="www.ilias.de">www.ilias.de</a><a href="https://www.ilias.de">www.ilias.de</a>', 'Praesent fermentum tempor ♥ tellus <span>Aliquam erat volutpat<b><a href="www.ilias.de"><b>curabitur lacinia pulvinar nibh</b> www.ilias.de</a>nunc eleifend leo vitae magnawww.ilias.denunc rutrum turpis sed pede www.ilias.de phasellus ♥ diam<a href="www.ilias.de">www.ilias.de</a>www.ilias.de'],
+            'test with properties' => ['<a href="https://www.ilias.de">www.ilias.de</a> <a purus="pretium" href="www.ilias.de" tellus="bibendum">www.ilias.de</a>', 'www.ilias.de <a purus="pretium" href="www.ilias.de" tellus="bibendum">www.ilias.de</a>'],
+            'test with tabs' => ['Mauris ac felis vel velit tristique imperdiet <a	href="www.ilias.de">www.ilias.de</a>', 'Mauris ac felis vel velit tristique imperdiet <a	href="www.ilias.de">www.ilias.de</a>'],
+            'test example input with newlines' => ["Das ist eine URL: <a href=\"https://www.ilias.de\">https://www.ilias.de</a>\nDas ist sogar ein Link: <a href=\"https://www.ilias.de\">https://www.ilias.de</a>\nDas ist ein Link hinter einem Wort: <a href=\"https://www.ilias.de\">Link</a> und noch mehr Text.", "Das ist eine URL: https://www.ilias.de\nDas ist sogar ein Link: <a href=\"https://www.ilias.de\">https://www.ilias.de</a>\nDas ist ein Link hinter einem Wort: <a href=\"https://www.ilias.de\">Link</a> und noch mehr Text."],
+            'test without protocol to URL with protocol' => ['<a href="https://www.ilias.de">www.ilias.de</a>', 'www.ilias.de'],
+            'test link with parameters' => ['<a href="http://ilias.de/ilias.php?ref_id=29&admin_mode=settings&cmd=view&cmdClass=ilobjdashboardsettingsgui&cmdNode=1c:lg&baseClass=ilAdministrationGUI">http://ilias.de/ilias.php?ref_id=29&admin_mode=settings&cmd=view&cmdClass=ilobjdashboardsettingsgui&cmdNode=1c:lg&baseClass=ilAdministrationGUI</a>', 'http://ilias.de/ilias.php?ref_id=29&admin_mode=settings&cmd=view&cmdClass=ilobjdashboardsettingsgui&cmdNode=1c:lg&baseClass=ilAdministrationGUI'],
+            'test multibyte' => ['I♥ILIAS', 'I♥ILIAS'],
```

Comment:
Could you change this test case (or add another one) to something like `I♥ILIAS' <a href ...>...</a> I♥ILIAS`.